LibPretixSync/Models/OrderPosition.swift

```diff
@@ -125,8 +125,9 @@ public struct OrderPosition: Model {
 
         // Check for open Questions
         let answerQuestionIDs = answers.map { return $0.question }
-        let unansweredQuestions = questions.filter { return !answerQuestionIDs.contains($0.identifier) }.filter { $0.isRequired }
-        if unansweredQuestions.count > 0 {
+        let unansweredQuestions = questions.filter { return !answerQuestionIDs.contains($0.identifier) }
+        let requiredUnansweredQuestions = unansweredQuestions.filter { $0.isRequired }
+        if requiredUnansweredQuestions.count > 0 {
```

Comment:
``if requiredUnansweredQuestions.count > 0`` will never show any questions if there is only one optional question, or am I wrong? If there is only an unanswered optional question, it should still be shown (once, if it is kept empty, it should be okay).

Reply:
Ah damn you're right! On it

Reply:
I'm unclear how exactly you do that on the server. For now, I'l just check if an `answers` list has been passed at all and if not, return the questions.